src/brain/apis/getProgress.ts

```diff
@@ -0,0 +1,82 @@
+import { Octokit } from '@octokit/rest';
+
+export const OWNERSHIP_FILE_LINK = "https://github.com/getsentry/sentry/blob/master/src/sentry/apidocs/api_ownership_stats_dont_modify.json";
+
+function get_emoji_for_type(type, api_rate) {
+  switch (type) {
+    case "public":
+      return api_rate == 0 ? ":sad_blob:" : api_rate < 20 ? ":blob-unamused:" : api_rate > 50 ? ":party-sunglasses-blob:" : "";
+    case "private":
+      return "";
+    case "unknown": case "experimental":
+      return api_rate == 0 ? ":party-sunglasses-blob:" : api_rate > 50 ? ":sad_blob:" : ":blob-unamused:";
+  }
+  return "";
+}
+
+function get_message_line_for_type(team_data, type, total) {
+  const api_rate = Math.round(team_data[type].length* 100/total)
+  const emoji = get_emoji_for_type(type, api_rate)
+  return `• ${type}: ${team_data[type].length} (${api_rate}%) ${emoji} \n`
+}
+
+function get_message_for_team(ownership_data, team) {
+  const team_data = ownership_data[team];
+  const total = 
+      team_data["public"].length + 
+      team_data["private"].length + 
+      team_data["experimental"].length + 
+      team_data["unknown"].length; 
+  return {
+    message: `Publish status for ${team} APIs:\n` +
+      get_message_line_for_type(team_data, 'public', total) + 
+      get_message_line_for_type(team_data, 'private', total) +
+      get_message_line_for_type(team_data, 'experimental', total) + 
+      get_message_line_for_type(team_data, 'unknown', total),
+    should_show_docs: team_data["unknown"].length + team_data["experimental"].length > 0,
+    review_link: `${OWNERSHIP_FILE_LINK}#L${team_data["block_start"]}`,
+  }
+}
+
+export default async function getProgress(team: string) {
+  const octokitWithToken = new Octokit({
```

Comment:
@mattgauntseo-sentry I need a file here not a commit. How do I get to a certain file from there?

Reply:
This could be named slightly better, but `GETSENTRY_ORG.api` is an octokit client configured with a token / app, so here you could do something like this:

```
const octokitWithToken = GETSENTRY_ORG.api;
```

And the rest of your code should work (Untested but hopefully that should be enough to unblock you).

For testing locally, if you set `FORCE_USER_TOKEN_GITHUB_CLIENT=true` in your `.env` file, the octokit client will be a personal token but will be an app in prod.